__tests__/gpg.test.ts

```diff
@@ -0,0 +1,56 @@
+import path = require('path');
+import io = require('@actions/io');
+import exec = require('@actions/exec');
+
+jest.mock('@actions/exec', () => {
+  return {
+    exec: jest.fn()
+  };
+});
+
+const tempDir = path.join(__dirname, 'runner', 'temp');
+process.env['RUNNER_TEMP'] = tempDir;
+
+import gpg = require('../src/gpg');
+
+describe('gpg tests', () => {
+  beforeEach(async () => {
+    await io.mkdirP(tempDir);
+  }, 300000);
```

Comment:
A timeout of 300 seconds seems way too high 🤔 same thing for `afterall`. The default is 5sec so I think it's fine to keep it at that: https://jestjs.io/docs/en/api#beforeallfn-timeout